plugins/hypervisors/kvm/src/main/java/org/apache/cloudstack/kvm/ha/KVMHostActivityChecker.java

```diff
@@ -213,6 +278,18 @@ protected boolean verifyActivityOfStorageOnHost(HashMap<StoragePool, List<Volume
         return poolVolMap;
     }
 
+    private boolean isHostServedByNfsPool(Host agent) {
+        List<StoragePoolHostVO> storagesOnHost = storagePoolHostDao.listByHostId(agent.getId());
+        for (StoragePoolHostVO storagePoolHostRef : storagesOnHost) {
+            StoragePoolVO storagePool = this.storagePool.findById(storagePoolHostRef.getPoolId());
+            if(Storage.StoragePoolType.NetworkFilesystem == storagePool.getPoolType()
+                    || Storage.StoragePoolType.ManagedNFS == storagePool.getPoolType()) {
```

Comment:
The collection cited in https://github.com/apache/cloudstack/pull/4978/files#r629568449 could be used here too.